chapter01/n02.py

```diff
@@ -0,0 +1,9 @@
+# 「パトカー」＋「タクシー」の文字を先頭から交互に連結して文字列「パタトクカシーー」を得よ．
+
+words = ''
+
+# zip関数を用いてforループの中で複数の要素を同時に取得し使う
+for w1, w2 in zip("パトカー", "タクシー"):
+    words += (w1 + w2)
+
```

Comment:
内包表記で書いたほうが読みやすいしこともあるから覚えておいてな
word変数を''で初期化する必要もなくなる。
```suggestion
word = ''.join([s1 + s2 for s1, s2 in zip('パトカー', 'タクシー')])
```